dask_geopandas/geohash.py

```diff
@@ -0,0 +1,224 @@
+import numpy as np
```

Comment:
```suggestion
"""
Geohash implementation

The code is originally based on the neathgeohash package, Copyright (c) 2020 Marek Dwulit, MIT License
(https://pypi.org/project/neathgeohash/#description).
That implementation is in turn based on "Geohash in Golang Assembly" blog (https://mmcloughlin.com/posts/geohash-assembly).

"""
import numpy as np
```

I would add something like this at the top of the file (copying code from an MIT licensed package is fine, but so referencing its copyright is the expected thing to do, I think (https://choosealicense.com/licenses/mit/)), and then you can remove the reference to the package and blog post elsewhere (now it's mentioned in multiple docstrings)